src/prpy/ik_ranking.py

```diff
@@ -53,13 +53,26 @@ def JointLimitAvoidance(robot, ik_solutions):
 
 
 class NominalConfiguration(object):
-    def __init__(self, q_nominal):
+    def __init__(self, q_nominal, max_deviation=2*numpy.pi):
```

Comment:
I think this default will break on robots that have revolute joints with more than 360 degrees of rotation. This is the case on ADA, along with any other robots, e.g. Robonaut, that don't use slip rings. Is it ever possible that it is feasible to move to a configuration that is further away, but has the same end-effector pose?

If we're not convinced that this can never happen, I think the `max_deviation` parameter should default to infinity.